library/Respect/Relational/Mapper.php

```diff
@@ -426,31 +445,33 @@ protected function fetchSingle(
         if ($this->typable($collection)) {
             $entityName = $this->inferGet($row, $this->getType($collection));
         }
-      
+
         $entities = new SplObjectStorage();
         $entities[$this->transformSingleRow($row, $entityName)] = $collection;
 
         return $entities;
     }
-    
+
     protected function getNewEntityByName($entityName)
     {
         $entityName = $this->getStyle()->styledName($entityName);
         $entityClass = $this->entityNamespace . $entityName;
         $entityClass = class_exists($entityClass) ? $entityClass : '\stdClass';
-        return new $entityClass;
+        $entityReflection = new ReflectionClass($entityClass);
+
+        return $entityReflection->newInstanceWithoutConstructor();
```

Comment:
The build is failing because `newInstanceWithoutConstructor()` is not available on PHP 5.3... :cry: